framework/src/modules/chain/logic/block.js

```diff
@@ -265,10 +251,7 @@ class Block {
  * @todo Add description for the params
  */
 __private.getAddressByPublicKey = function(publicKey) {
-	const publicKeyHash = crypto
-		.createHash('sha256')
-		.update(publicKey, 'hex')
-		.digest();
+	const publicKeyHash = hash(publicKey, 'hex');
```

Comment:
`hashedPublicKey`? that way its more consistent with others like `hashedBlock`, `hashedData`